### PR TITLE
mpes proposal

### PR DESCRIPTION
**Pull request by sanbrock**

### CODE REVIEW

**Comment by sanbrock** [comment]
https://fairmat-experimental.github.io/nexus-fairmat-proposal/d122a69ce0c953805e60a662e9580ee2c4a6fae7/index.html

**Review by mkoennecke — COMMENTED** [review]
NXdeflector.nxdl.xml

It is not needed to copy the content of NXtransformations into the definition. It is enough to state that a group of type NXtransformations is required.

But a depends_on field is missing which points to the start of the transformation chain.

Both NXcalibration and NXdistortion contain a NXcollection for describing the procedure employed. I feel that NXprocess would be more appropriate here. 

It is not clear to me if NXelectronanalyser is a base class or an application definition. With  a few more extras it would look like an application definition. 

It should be made clearer that NXmpes, NXmpes_ARPES are at the application definitions whereas the other things are suggested new base classes. 

You also suggest extensions to NXsource, mostly adding to the enums for probes and types etc. I would separate this out, that is an easy thing for the NIAC to agree too.

File: contributed_definitions/NXdistortion.nxdl.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        </doc>
+    </field>
+    <field name="applied" type="NX_BOOLEAN">
+        <doc>
+             Has the distortion correction been applied?
+        </doc>
+    </field>
+    <field name="symmetry" type="NX_INT" units="NX_UNITLESS">
+        <doc>
+             For symmetry-guided distortion correction
```
Comment (prjemian) [inline contributed_definitions/NXdistortion.nxdl.xml:39]: consider making this a link:

```rst
:ref:`symmetry-guided distortion correction <https://www.sciencedirect.com/science/article/abs/pii/S0304399118303474?via%3Dihub>`
```

File: contributed_definitions/NXspindispersion.nxdl.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXspindispersion" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <doc>
+         Subclass of NXelectronanalyser to describe the spin filters in photoemission
+         experiments.
+    </doc>
+    <field name="spin_filter_type" type="NX_CHAR">
```
Comment (prjemian) [inline contributed_definitions/NXspindispersion.nxdl.xml:8]: `spin_` is a common prefix for all these fields.  Consider if you might drop it (for shorter names).

Comment (prjemian) [reply contributed_definitions/NXspindispersion.nxdl.xml:8]: As written, it is easier to locate in a search.

File: contributed_definitions/NXmpes.nxdl.xml [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+                    <group name="field_aperture" optional="true" type="NXaperture">
+                        <doc>
+                             The size and position of the field aperture inserted in the column. To add
+                             additional or other apertures use the APERTURE group of NXcollectioncolumn.
+                        </doc>
+                    </group>
+                    <group name="contrast_aperture" optional="true" type="NXaperture">
+                        <doc>
+                             The size and position of the contrast aperture inserted in the column. To add
+                             additional or other apertures use the APERTURE group of Nxcollectioncolumn.
```
Comment (prjemian) [inline contributed_definitions/NXmpes.nxdl.xml:101]: capitalize `NX` here in `NXcollectioncolumn`

**Review by prjemian — APPROVED** [review]
LGTM as contributed

File: contributed_definitions/NXmpes.nxdl.xml (hunk 3 above)
Comment (domna) [reply contributed_definitions/NXmpes.nxdl.xml]: Typo is corrected

File: contributed_definitions/NXdistortion.nxdl.xml (hunk 1 above)
Comment (domna) [reply contributed_definitions/NXdistortion.nxdl.xml]: Changed it to a proper link

**Review by domna — COMMENTED** [review]
All the requested points have been addressed in the later commits.

**Pull request merged**